### ISSUE TITLE
use axios with react-kanban

### ISSUE THREAD

**Issue by natdev** [lourenci/react-kanban#396]
please can you help i get columns from a nodejs api but i can't fetch them. 

`import React,{Component} from "react";
import Board from '@lourenci/react-kanban'
import '@lourenci/react-kanban/dist/styles.css'
import data from "./data.json";
import axios from "axios";

export default class Kanban extends Component {
    _isMounted = false;
    constructor(props) {
        super(props);

        this.state = {

            donnees: {columns : []}
        }
    }

 componentDidMount() {
        let user_id = localStorage.getItem('UserId');
            // console.log(res.data);
        if(this.props.match) {
            const {match: {params}} = this.props;
           /* console.log(params.id);*/

        axios.get('http://localhost:8000/'+ user_id +'/projects/'+ params.id+'/projects', {headers: {'Authorization': `${localStorage.getItem('UserToken')}`}}).then((response) => {
            const data_elem = new Array();
             response.data.map( (elem) =>{
                 //console.log(elem.id);
                 axios.get('http://localhost:8000/'+ user_id +'/projects/columns/'+elem.id+'/cards', {headers: {'Authorization': `${localStorage.getItem('UserToken')}`}}).then((res) => {

                         elem['cards'] = res.data;

                 });
                 data_elem.push(elem);

             });

               // console.log( data_elem)

              this.setState({donnees:{columns: data_elem}})

          });

        }

        }

    render() {

        return(<div>

       <Board initialBoard={this.state.donnees}  />
    </div>)
    }

}`

the response i get from my api: 

columns: Array(2)
0: {id: "10076966", title: "test", cards: Array(1)}
1: {id: "10077601", title: "another column", cards: Array(0)}

**Comment by lourenci** [comment]
H, @natdev. Sorry for the delay.

You are controlling the board yourself, you need to use the [controlled one](https://github.com/lourenci/react-kanban#-controlled-and-uncontrolled).

TL;DR: use `<Board>{this.state.yourBoard}</Board>`.
